### PR TITLE
ui.py: white text on graphs

### PR DESCRIPTION
**Pull request by ghost**
* improved readability through white text
* automatic size selection based on window size

### CODE REVIEW

**Comment by pd0wm** [comment]
Can you fix the static analysis checks? By writing to `HOR` you have to mark it as global explicitly. You can probably clean it up so it's only set once the window is available.

File: tools/replay/ui.py [hunk 1]
```diff
@@ -45,7 +41,14 @@ def ui_thread(addr, frame_address):
   pygame.font.init()
   assert pygame_modules_have_loaded()
 
-  if HOR:
+  disp_info = pygame.display.Info()
+  max_height = disp_info.current_h
+
+  hor_mode = os.getenv("HORIZONTAL") is not None
+  hor_mode = True if max_height < 960+300 else hor_mode
+  angle_scale = 5.0
```
Comment (pd0wm) [inline tools/replay/ui.py:49]: No need to change angle_scale

File: tools/replay/ui.py [hunk 2]
```diff
@@ -234,8 +237,8 @@ def ui_thread(addr, frame_address):
     SPACING = 25
 
     lines = [
-      info_font.render("ENABLED", True, GREEN if sm['controlsState'].enabled else BLACK),
-      info_font.render("BRAKE LIGHTS", True, RED if sm['carState'].brakeLights else BLACK),
+      info_font.render("ENABLED", True, GREEN if sm['controlsState'].enabled else WHITE),
```
Comment (pd0wm) [inline tools/replay/ui.py:240]: Let's leave the colors for enabled and brake. Now it's not immediately clear if white enable means enabled or not.

File: tools/replay/ui.py (hunk 1 above)
Comment (BraunPhilipp) [reply tools/replay/ui.py:49]: Appreciate the quick feedback 😅

**Comment by pd0wm** [comment]
The plots are much more readable! We should have done this earlier :)

**Comment by BraunPhilipp** [comment]
Noticed the readability issue and thought it would be a nice first PR 😉

**Comment by pd0wm** [comment]
You can ignore the macOS failure. Not related to your PR.

**Pull request merged**